timerdemo/init.lua

```diff
@@ -11,11 +11,11 @@ assert(s)
 
 -- our timer functions
 local mytimerfunc1 = function(elapsed)
-	print("fn1: ", elapsed)
+	print("fn1: ", minetest.get_us_time(), elapsed)
 end
 
 local mytimerfunc2 = function(elapsed)
-	print("fn2: ", elapsed)
+	print("fn2: ", minetest.get_us_time(), elapsed)
```

Comment:
Ok, yes, this is fine, and probably useful to folks.

Reply:
missed the anonymous function, do this one too.

Reply:
Fixed